examples/text_labels_and_annotations/arrow_demo.py

```diff
@@ -49,20 +38,14 @@ def make_arrow_plot(data, size=4, display='length', shape='right',
     alpha : float
         Maximum opacity of arrows.
     **kwargs
-        Can be anything allowed by a Arrow object, e.g. *linewidth* or
+        Can be anything allowed by an Arrow object, e.g. *linewidth* or
```

Comment:
Is there anyway to cross link to the ArrowStyle object here?

Reply:
linked